sensor_msgs/Image dualFisheye
---
sensor_msgs/Image equirectangular
